src/elements/form/Control.ts

```diff
@@ -99,7 +99,8 @@ import { Helpers } from './../../utils/Helpers';
             <div class="error-message">
                 <span *ngIf="isDirty && errors?.required">{{control.label | uppercase}} is required</span>
                 <span *ngIf="isDirty && errors?.minlength">{{control.label | uppercase}} is required to be a minimum of {{ control.minlength }} characters</span>
-                <span *ngIf="isDirty && errors?.maxlength">{{control.label | uppercase}} is required to be a maximum of {{ control.maxlength }} characters</span>
+                <span *ngIf="isDirty && maxLengthMet && !maxLength">Sorry, you have reached the maximum character count of {{ control.maxlength }} for this field</span>
```

Comment:
Don't need this error message. NG2 will automatically trigger validation on maxlength validator.